test/lit.cfg.py

```diff
@@ -74,6 +82,11 @@ def setDeviceFeatures(config, device, compiler):
 
     config.available_features.add("%s-%s" % (compiler, API))
 
+    HighestShaderModel = getHighestShaderModel(device["Features"])
+    if (6, 6) <= HighestShaderModel:
+        # https://github.com/microsoft/DirectX-Specs/blob/master/d3d/HLSL_ShaderModel6_6.md#derivatives
+        config.available_features.add("Derivatives")
```

Comment:
I would probably call this `DerivativesInCompute` since derivatives in graphics shaders are a different and more foundational feature.